### DISCUSSION TITLE
image looks fine, but recognition keeps tilting the picture

### DISCUSSION THREAD

**Discussion by jeroenvdd** [jomjol/AI-on-the-edge-device#2541] [Q&A]
Hi all,
I'm struggling with the image processing my image looks like this:
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/6975276/552e7ead-af22-4d18-ac04-dff153141dbe)
but the current 'detection' creates this:
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/6975276/ef48ec55-5927-4126-8256-4f762783f93e)

it has worked fine for a long while so any ideas what is causing / could be causing the drift?

**Comment by caco3** [comment]
Try to make the alignment marks smaller and futher away.

**Reply by jeroenvdd** [reply]
ok will do i tought the bigger the better since it has more 'to align' to

**Reply by caco3** [reply]
It it gets too large, the accuracy goes down, because internal it always gets resized to a fixed pixel size.
